### DISCUSSION TITLE
Solar Max Import

### DISCUSSION THREAD

**Discussion by mif11k** [dingo35/SmartEVSE-3.5#42] [Q&A]
I noticed that after upgrading to 3.5.0-RC1 I am not able to set maximal import current for solar charging below MinCurrent. I read in the release notes that  ImportCurrent >= StartCurrent + MinCurrent; many users misconfigured Solar mode.... I don't understand the intent. I use the solar mode to utilise the overproduction of the solar system only, that means I do not want any import while solar charging. This could be achieved by setting max import to zero in previous releases. Since 3.5.0-RC1 this cannot be achieved, because the charger adds energy from the grid always to the PV generated power even when the car can be charged by the solar power only. Any solution for my use case with the latest release?

**Comment by dingo35** [comment]
Is already solved in master, wait for RC2.
